auth-web/src/views/auth/SigninView.vue

```diff
@@ -39,13 +39,15 @@ export default class Signin extends Mixins(NextPageMixin) {
   @Prop({ default: '' }) redirectUrlLoginFail: string
 
   private async authenticationComplete () {
+    await this.loadUserInfo()
     // Check if user is authenticated, and redirect according to specified redirect
     // or fallback to default route for their login source
     if (this.$store.getters['auth/isAuthenticated']) {
       this.$root.$emit('signin-complete')
       if (this.redirectUrl) {
         this.redirectTo(decodeURIComponent(CommonUtils.isUrl(this.redirectUrl) ? this.redirectUrl : `/${this.redirectUrl}`))
       } else {
+        await this.syncUser()
```

Comment:
So I don't think this call is necessary.  We are emitting `signin-conplete` above, which tells App.vue to load user info and sync the user/account.  Are you sure this is what was causing the issue in NextPageMixin?

Reply:
Actually,  you're right that calling syncUser here does fix it.  There's a race condition between the handler for `signin-complete` in App.vue and the logic in this comopnent.  Unfortunately, this fix means we'll be syncing the user/account twice.  I'm going to take another look at this.

Reply:
Yes, thats needed,
because the NextPageMixin needs the value for membership and userterms, since the event is fired along with the redirecturl call, the delay for all the api calls makes those values missing in the NextPageMixin 
calling await syncUser solves that issue

Reply:
Yeah, I get that now.  The only thing is we'll be syncing the user twice.  I'm okay to merge this now, but I'll take a look at improving that flow.  We probably shouldn't emit an event to trigger syncing.